test/Microsoft.AspNet.Mvc.FunctionalTests/ModelBindingTests.cs

```diff
@@ -1327,5 +1327,42 @@ public class ModelBindingTests
             var body = await response.Content.ReadAsStringAsync();
             Assert.Equal(expectedContent, body);
         }
+
+        [Fact]
+        public async Task FormFileModelBinder_CanBind_SingleFile()
+        {
+            // Arrange
+            var server = TestServer.Create(_services, _app);
+            var client = server.CreateClient();
+            var url = "http://localhost/FileUpload/UploadSingle";
+            var streamContent = new StreamContent(new MemoryStream());
```

Comment:
empty stream? we should actually look at the contents of the file, and that requires some contents